sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample7_CopyCustomModel.md

```diff
@@ -62,8 +62,8 @@ Now that we have authorization from the target Form Recognizer resource, we exec
 string modelId = "<source_modelId>";
 CustomFormModelInfo newModel = await sourceClient.StartCopyModelAsync(modelId, targetCopyAuth).WaitForCompletionAsync();
 
-Console.WriteLine($"Original modelID => {modelId}");
-Console.WriteLine($"Copied modelID => {newModel.ModelId}");
+Console.WriteLine($"Original model ID => {modelId}");
+Console.WriteLine($"Copied model ID => {newModel.ModelId}");
```

Comment:
Just me being nitpicky. In general, calling it `ID` if it's a word by itself. Calling it `Id` if it's part of a larger name (e.g. `resourceId`).